pkg/scheduler/api/compatibility/compatibility_test.go

```diff
@@ -231,12 +233,12 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 				"BalancedResourceAllocation",
 				"SelectorSpreadPriority",
 				"NodeAffinityPriority",
-				"TaintTolerationPriority",
 				"InterPodAffinityPriority",
 			),
-			wantFilterPlugins: sets.NewString(
-				"TaintToleration",
-			),
+			wantPlugins: map[string][]kubeschedulerconfig.Plugin{
+				"FilterPlugin": {{Name: "TaintToleration"}},
```

Comment:
How should we sort the same type of plugin so that` cmp.Diff(tc.wantPlugins, gotPlugins)` does not get an error?
`cmp.Diff` should ignore the order of the elements in the slice, but it is not.

Reply:
Why do we need to sort? The order is stable, wouldn't for example NodePorts be always before taint toleration?

Reply:
> The order is stable, wouldn't for example NodePorts be always before taint toleration?

Thank you for your quickly reply. Where can I get this stable order.

Reply:
https://github.com/kubernetes/kubernetes/blob/c1a735c6420e30e16a1c685750eb692e5b28df56/pkg/scheduler/algorithm/predicates/predicates.go#L147

Reply:
But in general, just test it out and see where it lands in the list, it shouldn't change from run to run.

Reply:
Ok, Thanks.